### PR TITLE
⭐️ collect running kernel in sbom

### PR DESCRIPTION
**Pull request by chris-rock**

### CODE REVIEW

File: sbom/sbom.mql.yaml [hunk 1]
```diff
@@ -12,10 +12,16 @@ packs:
         mql: asset { name platform version arch ids labels cpes.map(uri) }
       - uid: mondoo-sbom-packages
         title: Retrieve list of installed packages
-        mql: packages { name version purl cpes.map(uri) arch origin format files.map(path) }
```
Comment (czunker) [inline sbom/sbom.mql.yaml]: When we remove this here, we also need to remove it in the backend.

Comment (chris-rock) [reply sbom/sbom.mql.yaml]: Nice catch, that is an accident. Just accelerated testing. Will remove.

**Comment by chris-rock** [comment]
@czunker ready for re-review

**Review by czunker — APPROVED** [review]
Thanks @chris-rock

**Pull request merged**